src/libstd/sys/unix/ext/raw.rs

```diff
@@ -15,6 +15,8 @@
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type uid_t = u32;
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type gid_t = u32;
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type pid_t = i32;
+#[unstable(feature = "pthread_t", issue = "0")]
+pub type pthread_t = usize;
```

Comment:
This is actually a little more involved, taking a look at the definition in liblibc:

```
$ git grep 'type pthread_t'
src/unix/bsd/mod.rs:pub type pthread_t = ::uintptr_t;
src/unix/notbsd/android/mod.rs:pub type pthread_t = c_long;
src/unix/notbsd/linux/mod.rs:pub type pthread_t = c_ulong;
```